CHANGELOG.md

```diff
@@ -17,6 +17,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Added `ModelUsage` to `ModelCard` class.
 - Added a way to specify extra parameters to the predictor in an `allennlp predict` call.
 - Added a way to initialize a `Vocabulary` from transformers models.
+- Added the ability to use `Predictors` with multitask models.
```

Comment:
```suggestion
- Added the ability to use `Predictors` with multitask models through the new `MultiTaskPredictor`.
```

Reply:
Oh did I forget to do that? I meant to. Sorry!